explorer/jest.setup.ts

```diff
@@ -1,16 +1,16 @@
-import { Connection } from 'typeorm'
+import { getConnection } from 'typeorm'
 import { openDbConnection } from './src/database'
 import { clearDb } from './src/__tests__/testdatabase'
 
 process.env.NODE_ENV = 'test'
 
-let globalDBConnection: Connection
 beforeAll(async () => {
-  globalDBConnection = await openDbConnection()
+  await openDbConnection()
 })
 afterEach(() => clearDb())
 afterAll(async () => {
-  if (globalDBConnection) {
-    await globalDBConnection.close()
+  const db = getConnection()
+  if (db) {
```

Comment:
The connection will always exist, or throw an exception.

Reply:
!!!!